test/functional/toolchains/cmake/cmakedeps2/test_cmakeconfigdeps_new.py

```diff
@@ -1572,3 +1572,43 @@ def build(self):
     assert "find_package(matrix)" in c.out
     assert "target_link_libraries(... matrix::matrix)" in c.out
     assert "Conan: Target declared imported INTERFACE library 'matrix::matrix'" in c.out
+
+
+@pytest.mark.tool("cmake", "3.27")
+def test_dependendecy_multi_filename():
+    tc = TestClient()
+    find_name = "foo"
+    conanfile = textwrap.dedent(f"""
+    from conan import ConanFile
+
+    class Pkg(ConanFile):
+        name = "pkg"
+        version = "0.1"
+        def package_info(self):
+            self.cpp_info.set_property("cmake_extra_file_names", ["{find_name}", "extra_name2"])
+    """)
+    cml = textwrap.dedent(f"""
+    cmake_minimum_required(VERSION 3.27)
+    project(app)
+    find_package({find_name} CONFIG REQUIRED)
```

Comment:
I don't get this. Shouldn't it be ``find_package("extra_name2", ...)``?

Reply:
Ok, now I get this, the ``extra_name2`` is not used, but only ``foo`` which is already an extra name.
I'd susggest a simpler:
```
self.cpp_info.set_property("cmake_extra_file_names", ["extra_name"])
```
No prob with hardcoding ``extra_name`` everywhere in the test, it is nice and explicit, more meaningful than ``foo``